### PR TITLE
{bio}[GCCcore/9.3.0] Bracken v2.6.0

### PR DESCRIPTION
**Pull request by sassy-crick**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/b/Bracken/Bracken-2.6.0-GCCcore-9.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+checksums = [
+    'fb1837d6f32b8f8c87353b9dc8a23d3418c348b00824a7064eb58f9bab11ea68',  # v%(version)s.tar.gz
+]
+
+builddependencies = [('binutils', '2.34')]
+
+# no need to build in parallel 
+parallel = 1
+
+build_cmd = "cd %(builddir)s/%(name)s-%(version)s/src && make"
```
Comment (smoors) [inline easybuild/easyconfigs/b/Bracken/Bracken-2.6.0-GCCcore-9.3.0.eb:45]: @sassy-crick no need to change the build_cmd here, just update start_dir:
```suggestion
start_dir = 'src'
```

Comment (sassy-crick) [reply easybuild/easyconfigs/b/Bracken/Bracken-2.6.0-GCCcore-9.3.0.eb]: Thanks for pointing this out, that was something I did not know (or missed). I have updated the PR.

**Comment by smoors** [comment]
Test report by @smoors
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node378.hydra.os - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz (skylake_avx512), Python 2.7.5
See https://gist.github.com/d089f9399ca04cf9e3ecd6d746a2cae4 for a full test report.

**Comment by smoors** [comment]
Going in, thanks @sassy-crick!

**Pull request merged**